### PR TITLE
Model nulls normally when tabular preset has constraints

### LINKED ISSUE #763: Tabular Preset with constraints should handle nulls correctly
### Problem Description

SDV Tabular preset should handle constraints correctly when the constraints add null values. Currently, if I use a CustomConstraint that specifies a certain column should have null values based on a custom formula, those null values get overwritten.

### PR DESCRIPTION
**Pull request by katxiao**
Resolves #763 

If tabular preset has constraints, model nulls normally instead of pre- and post-processing the data. Also, update the preset name.

### CODE REVIEW

File: sdv/lite/tabular.py [hunk 1]
```diff
@@ -86,13 +91,14 @@ def fit(self, data):
...
                 Data to fit the model to.
         """
-        self._null_percentages = {}
-
-        for column, column_data in data.iteritems():
-            num_nulls = column_data.isna().sum()
-            if num_nulls > 0:
-                # Store null percentage for future reference.
-                self._null_percentages[column] = num_nulls / len(column_data)
+        if not self._null_column:
```
Comment (amontanez24) [inline sdv/lite/tabular.py:94]: can we add the test case for when this should not learn the null percentages?

**Review by amontanez24 — APPROVED** [review]
Just one comment about adding test cases. Otherwise LGTM!

**Review by pvk-developer — APPROVED** [review]

File: sdv/lite/tabular.py (hunk 1 above)
Comment (katxiao) [reply sdv/lite/tabular.py:94]: @amontanez24 That should be covered in the `test_fit` unit test, and I also assert that null percentages is null at the end.

Comment (katxiao) [reply sdv/lite/tabular.py:94]: Added!
